src/init/features/genkit.ts

```diff
@@ -0,0 +1,66 @@
+import * as spawn from "cross-spawn";
+import { logger } from "../../logger";
+import { doSetup as functionsSetup } from "./functions";
+import { Options } from "../../options";
+import { Config } from "../../config";
+import { prompt } from "../../prompt";
+
+/**
+ * doSetup is the entry point for setting up the genkit suite.
+ */
+export async function doSetup(setup: any, config: Config, options: Options): Promise<void> {
+  if (setup.functions?.languageChoice !== "typescript") {
+    await prompt(setup, [
+      {
+        name: "continueFunctions",
+        type: "confirm",
+        message:
+          "Genkit with Firebase uses Cloud Functions for Firebase with TypeScript. Initialize Functions to continue?",
+        default: true,
+      },
+    ]);
+    if (!setup.continueFunctions) {
+      logger.info("Stopped Genkit initialization");
+      return;
+    }
+
+    // Functions with genkit should always be typescript
+    setup.languageOverride = "typescript";
+    await functionsSetup(setup, config, options);
```

Comment:
Can you test how this behaves when initing multiple products at once? Particularly, i'd be concerned about initing gen kit + python or javascript functions

Reply:
Yep -- it works in that case. Both if the directory already exists and if you select both functions and genkit from the init menu. Functions seems to always init first but I've added a `delete setup.languageOverride` to ensure it still works even in reverse